test-galasactl-ecosystem.sh

```diff
@@ -484,12 +593,13 @@ runs_get_check_details_format_output  $RUN_NAME
 runs_get_check_raw_format_output  $RUN_NAME
 
 # Query the result with the age parameter 
-runs_get_check_summary_format_output_with_valid_age_param $RUN_NAME
+runs_get_check_raw_format_output_with_from_and_to $RUN_NAME
+runs_get_check_raw_format_output_with_just_from $RUN_NAME
 
 # Check that the age parameter throws correct errors with invalid values
-runs_get_check_summary_format_output_with_invalid_age_param $RUN_NAME
-runs_get_check_summary_format_output_with_no_runname_and_no_age_param
-runs_get_check_summary_format_output_with_badly_formed_age_param
+runs_get_check_raw_format_output_with_no_runname_and_no_age_param
+runs_get_check_raw_format_output_with_invalid_age_param
+runs_get_check_raw_format_output_with_older_to_than_from_age
```

Comment:
I think all the variations in terms of syntax are already verified in the junits.

I think at the IVT level we want to test that the results match up to what we thing they should be... but in the case of this age parameter, we can't do any before-after testing really, as the results are going to be time-dependent. 

So... I think what you've done is ok, though the older-than... and invalid-age tests aren't needed, given that we cover those in unit tests ? But it probably doesn't hurt to leave them there anyway.